### ISSUE TITLE
Problem when kniting to gitbook: pandoc-citeproc: reference xyz not found

### ISSUE THREAD

**Issue by dennisgrobe** [ismayc/thesisdown#77]
Hello ismayc

First thank you for this amazing template and that you share it with us! I've learned a lot writing my thesis with this setup and I'm now able to produce really fancy looking documents.

Creating PDF's works totally fine without any error message.

But when I want to **knit to gitbook**, I get some error messages and the biggest problem is, that the references are not implemented.
I get following error messages:

`[WARNING] Could not parse YAML metadata at line 37 column 1: leading empty lines contain more spaces than the first non-empty line in scalar: " \n" `

`pandoc-citeproc: reference xyz not found`

`[WARNING] This document format requires a nonempty <title> element.
  Defaulting to 'thesis.utf8' as the title.
  To specify a title, use 'title' in metadata or --metadata title="...".`

Can you help me with this issue?

Thanks, Dennis

**Comment by ismayc** [comment]
Unfortunately, I'm not able to reproduce, but my best bet for you to try to figure out what is going on is to comment out lines at the top of the YAML corresponding to things like the `abstract` in the `index.Rmd` file. The index.Rmd file is optimized for the PDF build and can sometimes be finicky for gitbook.

**Comment by dennisgrobe** [comment]
Thank you for your answer and sorry for my late reply. I tried your tips and a lot of other things.
Unfortunately none of my efforts lead to a solution. 

Interestingly the problem even appears when knitting your template (that I didn't edit) to gitbook. pandoc-citeproc seems to have problems to find (or read?) the bibliography file. 
All my software is running on the newest version. the problem may has arised with the update of pandoc... 

Would it help if I shared my directory with you???

PS: I would fully understand, if this issue is not a priority to you!

Kind regards, Dennis
